DayrSuryan/DSEthiop13.xml

```diff
@@ -5,7 +5,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title>Maṣḥafa zǝq</title>
```

Comment:
I think the standard title of the work is: "Maṣḥafa ziq"

Reply:
Yes, I think you looked again at an outdated version, it was already corrected in the PR

Reply:
I actually corrected to "The Book of Ziq", as the title of the description.